src/components/dev-dot-content/mdx-components/mdx-inline-code/mdx-inline-code.module.css

```diff
@@ -0,0 +1,9 @@
+.inlineCode {
+	background-color: var(--token-color-palette-neutral-50);
+	border-radius: 5px;
+	border: 1px solid rgba(101, 106, 118, 0.2);
+	font-size: 0.8125rem;
+	font-weight: 400;
+	line-height: 16px;
+	padding: 2px 4px;
+}
```

Comment:
Future 📌

We need to update some of these values to use HDS tokens, so I create a ticket to capture that work: [[MdxInlineCode] Update CSS to use hds tokens](https://app.asana.com/0/1202097197789424/1202578369060897/f) 🎟️

Reply:
🙌 nice, ty!